codespell_lib/data/dictionary.txt

```diff
@@ -1795,6 +1826,17 @@ arugments->arguments
 arund->around
 asbtraction->abstraction
 asceding->ascending
+asend->ascend
+asendance->ascendance
+asendancey->ascendancy
+asendancy->ascendancy
+asended->ascended
```

Comment:
also "as ended"

Reply:
I can't think of a sentence this would sound grammatically correct in

Reply:
Random example: "Youtube doesn't recognize the stream as ended"

Reply:
Still doesn't sound correct to me.

Reply:
Uhm. Well, I'm not a native speaker. Please correct me if I'm wrong.

But...

Google yields 500.000 hits for "as ended". Please try yourself.

Another example that sounds reasonably to me: "When the Soviet Union broke down in 1991, Ukraine stated its freedom as well as ended up being a complimentary condition."

Reply:
fair enough. Added in 2a11b60